iothub/device/src/Transport/RetryDelegatingHandler.cs

```diff
@@ -11,51 +11,47 @@
 
 namespace Microsoft.Azure.Devices.Client.Transport
 {
-    using ExponentialBackoff = Microsoft.Azure.Devices.Client.ExponentialBackoff;
-
     internal class RetryDelegatingHandler : DefaultDelegatingHandler
     {
         // RetryCount is used for testing purpose and is equal to MaxValue in prod.
-        private static int s_retryMaxCount = int.MaxValue;
+        private const int RetryMaxCount = int.MaxValue;
```

Comment:
shouldn't this be _camelCase?

Reply:
Const are TitleCase. I never liked that standard, but that is what most teams seem to go with.